src/components/shared/index.ts

```diff
@@ -2,9 +2,8 @@ export * from './Button';
 export * from './ButtonGroup';
 export * from './Card';
 export * from './DropDown';
-export * from './modals/Modal';
```

Comment:
Why are you removing this? this is the work of @shyakadavis where he was exporting this modal